django/db/backends/oracle/utils.py

```diff
@@ -54,6 +54,7 @@ def from_datetime(cls, dt):
 
 class BulkInsertMapper:
     BLOB = 'TO_BLOB(%s)'
+    CBLOB = 'TO_CLOB(%s)'
```

Comment:
Am late to the review party - should the constant be `CLOB` instead of `CBLOB`?

Reply:
Good catch, thanks. I will fix this

Reply:
Fixed in 67e7dffe9543aff259f63c8f12d15642fe7be100.

Reply:
Sorry about that, thanks!